bin/v-update-sys-hestia-git

```diff
@@ -172,287 +211,133 @@ if [ -z "$branch" ]; then
     echo "ERROR: No branch detected."
     exit
 fi
+REPO="$fork/hestiacp"
+# Forward slashes in branchname are replaced with dashes to match foldername in github archive.
+branch_dash=$(echo "$branch" |sed 's/\//-/g');
 
 #----------------------------------------------------------#
 #                       Action                             #
 #----------------------------------------------------------#
 
-# Install needed software
-echo "[*] Updating APT package cache..."
-apt-get -qq update > /dev/null 2>&1
-echo "[*] Checking for missing dependencies and installing required libraries..."
-apt-get -qq install -y $SOFTWARE > /dev/null 2>&1
-
-# Fix for Debian PHP Envroiment
-if [ $BUILD_ARCH == "amd64" ]; then
-    if [ ! -L /usr/local/include/curl ]; then
-        ln -s /usr/include/x86_64-linux-gnu/curl /usr/local/include/curl
-    fi
-fi
-
-# Get system cpu cores
-NUM_CPUS=$(grep "^cpu cores" /proc/cpuinfo | uniq |  awk '{print $4}')
-
-# Check for existence of flags argument and set packages to build
-if [ -n "$flags" ]; then
-    if [ "$flags" = "all" ]; then
-        HESTIA_B='true'
-        NGINX_B='true'
-        PHP_B='true'
-    fi
+    # Install needed software
+if [ "$OSTYPE" = 'rhel' ]; then
+    # Set package dependencies for compiling
+    SOFTWARE='gcc gcc-c++ make libxml2-devel zlib-devel libzip-devel gmp-devel libcurl-devel gnutls-devel unzip openssl openssl-devel pkg-config sqlite-devel oniguruma-devel rpm-build wget tar git curl perl-IPC-Cmd'
+
+    echo "Updating system DNF repositories..."
+    dnf install -y -q 'dnf-command(config-manager)'
+    dnf install -y -q dnf-plugins-core
+    dnf config-manager --set-enabled powertools > /dev/null 2>&1
+    dnf config-manager --set-enabled PowerTools > /dev/null 2>&1
+    dnf upgrade -y -q
+    echo "Installing dependencies for compilation..."
+    dnf install -y -q $SOFTWARE
```

Comment:
We won't need any of this maybe unzip, rpm-build, wget, tar, curl as we are not building hestia-nginx and hestia-php.